src/MultiTypeSymbolTable.chpl

```diff
@@ -6,6 +6,7 @@ module MultiTypeSymbolTable
     use Reflection;
     use ServerErrors;
     use Logging;
+    use Regex;
```

Comment:
So the `Regex` module changed names between chapel version 1.24 (`Regexp`) and 1.25 (`Regex`). If you change this to `use ArkoudaRegexCompat;` it should do the right import depending on the version

Reply:
Updated

Reply:
I do not have a ticket against Arkouda for this (just an internal one for our sprint plans.) Feel free to create one for it